src/test/ledger/PathSet.h

```diff
@@ -22,7 +22,7 @@
 
 #include <ripple/basics/Log.h>
```

Comment:
What do you think about moving `PathSet.h` to `src/test/support/`? It looks looks like it is used for `app` and `ledger` tests.